### ISSUE TITLE
can it support path contain system variable?

### ISSUE THREAD

**Issue by Freed-Wu** [weirongxu/coc-explorer#146]
such as `:CocCommand explorer $HOME`?

or allow user use \` to escape, like defx, `:Defx \`expand('%:p:h')\` `, netrw, `:split \`echo $HOME\` `?

**Comment by wookayin** [comment]
Why not just `:CocCommand explorer %:p:h`? Isn't it consistent with `:e %:p:h`, etc.? The ` syntax seems surprising and not standard.

**Comment by weirongxu** [comment]
> Why not just `:CocCommand explorer %:p:h`? Isn't it consistent with `:e %:p:h`, etc.? The ` syntax seems surprising and not standard.

You are right, but we need to support expressions and variables, I think supporting \` is still necessary, maybe I should support two usages at the same time.

**Comment by wookayin** [comment]
Environment variables might be fine; it's the feature we can define, but in my humble opinion, vim functions are exactly for this in general.

**Comment by Seybo** [comment]
@wookayin is your suggestion supposed to work? `:CocCommand explorer %:p:h`
Trying to use it but it gives me an error `[coc.nvim] Error: ENOENT: no such file or directory, scandir '%:p:h'`
Am i missing something?

**Comment by wookayin** [comment]
It is not implemented, as you can see if you read the discussion carefully.

**Comment by Freed-Wu** [comment]
> UPDATE: Actually I didn't know the built-in command :e `echo 1` works (wow!). Given that backticks works for builtins, maybe it would be not a bad idea to support such syntax.

As an addition, according to `:help backtick-expansion`, this feature cannot cross all platforms because it only support unix-like OS.

**Comment by sumnerevans** [comment]
Workaround to open coc-explorer in the current working directory as set by Vim. Good for use with [vim-rooter](https://github.com/airblade/vim-rooter):

```
function! CocExploreCwd()
    let cwd = substitute(execute(":pwd"), '\n', '', '')
    exe 'CocCommand explorer ' . cwd
endfunction
map <S-T> :call CocExploreCwd()<CR>
```
